lib/flipper/railtie.rb

```diff
@@ -0,0 +1,21 @@
+module Flipper
+  class Railtie < Rails::Railtie
+    config.before_configuration do
+      config.flipper = ActiveSupport::OrderedOptions.new
+      config.flipper.memoizer = ActiveSupport::OrderedOptions.new
+      config.flipper.memoizer.preload = true
+    end
+
+    initializer "flipper.memoizer" do |app|
+      if config.flipper.memoizer
+        app.middleware.use Flipper::Middleware::Memoizer, config.flipper.memoizer
+      end
```

Comment:
@jnunemaker I'd love your thoughts on configuration.

We already have `Flipper::Configuration`, and this PR will introduce `Rails.application.config.flipper`. I'm tempted to combine them, but I can't decide which direction to go. I'm not sure it makes sense to add `memoizer` and `cloud` to `Flipper::Configuration` since they won't get used outside of Rails.

Would it make sense to deprecate `Flipper::Configuration`?

```ruby
Rails.application.configure do
  config.flipper.default = -> { Flipper.new(…) }
end
```

Outside of Rails, you'd be on your own for managing configuration and you would set `Flipper.instance` manually.

🤔

Reply:
My preference is to have everything in flipper and rails use flipper stuff. I don't like tying to rails anymore than is convenient. That is just my quick gut reaction. I can read this more today or tomorrow and see if I feel the same.

Reply:
Ok, I'll take a pass at moving everything to Flipper and see how it feels.